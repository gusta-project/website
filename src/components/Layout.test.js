import React from 'react';
import renderer from 'react-test-renderer';

import { PureLayout } from './Layout';

jest.mock('components/Header', () => require('utils').mockComponent('Header'));
jest.mock('components/Footer', () => require('utils').mockComponent('Footer'));

describe('Layout', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <PureLayout
          data={{
            site: {
              siteMetadata: {
                title: 'Site',
                company: 'Company'
              }
            }
          }}
        >
          Testing
        </PureLayout>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
