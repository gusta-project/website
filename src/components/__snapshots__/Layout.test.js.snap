// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
Array [
  <Header />,
  <div
    className="container"
  >
    <main>
      Testing
    </main>
    <Footer />
  </div>,
]
`;
